src/client/entity-editor/common/sort-name-field.tsx

```diff
@@ -166,7 +166,7 @@ function SortNameField({
 			label={label}
 			ref={(node) => { input = node; }}
 			tooltipText="Alphabetical sorting name. Examples: 'Dickens, Charles', 'Christmas Carol, A'.
-			<br>You can try to fill it automatically with the guess button"
+			You can try to fill it automatically with the guess button"
```

Comment:
This should not be part of this PR, you'll need to revert this change (and open another PR if you want to fix it ;) )

Reply:
Sure thing!